packages/openapi3/test/models.test.ts

```diff
@@ -0,0 +1,769 @@
+import { expectDiagnostics } from "@cadl-lang/compiler/testing";
```

Comment:
Is this all moved from openapi-output.ts or are there interesting changes I should be reviewing?

Reply:
this test `it("uses json name is specified",` is new